Lib/ast.py

```diff
@@ -1273,16 +1285,12 @@ def _write_fstring_inner(self, node):
 
     def visit_FormattedValue(self, node):
         def unparse_inner(inner):
-            unparser = type(self)(_avoid_backslashes=True)
+            unparser = type(self)(_avoid_backslashes=False)
```

Comment:
Do we still need this mode or can we drop `avoid_backslashes` all together?

Reply:
I guess we can drop it.